charts/portworx/v2.8.1/templates/portworx-csi.yaml

```diff
@@ -65,32 +117,68 @@ spec:
       containers:
         - name: csi-external-provisioner
           imagePullPolicy: Always
-          image: "{{ template "px.getcsiProvisioner" . }}/csi-provisioner:v0.2.0"
+          image: "docker.io/openstorage/csi-provisioner:v1.6.1-1"
+          args:
+            - "--v=3"
+            - "--provisioner=pxd.portworx.com"
+            - "--csi-address=$(ADDRESS)"
+            - "--enable-leader-election"
+            - "--leader-election-type=leases"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+          securityContext:
+            privileged: true
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+        - name: csi-snapshotter
+          imagePullPolicy: Always
+          image: k8s.gcr.io/sig-storage/csi-snapshotter:v4.0.0
           args:
-            - "--v=5"
-            - "--provisioner=com.openstorage.pxd"
+            - "--v=3"
             - "--csi-address=$(ADDRESS)"
+            - "--leader-election=true"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+          securityContext:
+            privileged: true
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+        - name: csi-snapshot-controller
+          imagePullPolicy: Always
+          image: k8s.gcr.io/sig-storage/snapshot-controller:v4.0.0
+          args:
+            - "--v=3"
+            - "--leader-election=true"
           env:
             - name: ADDRESS
               value: /csi/csi.sock
+          securityContext:
+            privileged: true
           volumeMounts:
             - name: socket-dir
               mountPath: /csi
-        - name: csi-attacher
+        - name: csi-resizer
           imagePullPolicy: Always
-          image: "{{ template "px.getcsiImages" . }}/csi-attacher:v0.2.0"
+          image: k8s.gcr.io/sig-storage/csi-resizer:v1.1.0
```

Comment:
Latest resizer is 1.3.0
https://github.com/portworx/px-installer/blob/1494d469cd1a6bb7032911e8a7ccabc477b288f0/px-spec-websvc/csi.go#L280